project/Dependencies.scala

```diff
@@ -15,12 +15,14 @@ object Dependencies {
   val sparkSql = "org.apache.spark" %% "spark-sql" % sparkVersion % "provided"
 
   val sparkBigQuery = "com.google.cloud.spark" %% "spark-bigquery-with-dependencies" % "0.17.1"
```

Comment:
Can we go to latest? https://github.com/GoogleCloudDataproc/spark-bigquery-connector/releases/tag/0.18.0

Reply:
Done